modules/functional_expansion_tools/src/series/CompositeSeriesBasisInterface.C

```diff
@@ -10,6 +10,8 @@
 #include "CompositeSeriesBasisInterface.h"
 #include "Legendre.h"
 
+#include "libmesh/auto_ptr.h"
```

Comment:
Again - it's unclear why this is here. I assume something downstream...

Reply:
`libmesh_make_unique`